roles/apache/tasks/ssl.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: Create cert_dir if not exists
+  file: 
+    dest: "{{cert_dir}}"
+    state: directory
+    owner: root
+    group: root 
+    mode: 0755
+
+- name: Add ssl certificate chain file
+  copy:
+    src: private_files/{{inventory_hostname}}/{{httpd.chain}}
+    dest: "{{cert_dir}}/{{httpd.chain}}"
+  when: httpd.chain is defined
+
+- name: Add ssl certificate file
+  copy:
+    src: private_files/{{inventory_hostname}}/{{httpd.cert}}
+    dest: "{{cert_dir}}/{{httpd.cert}}"
+  when: httpd.cert is defined
+
+- name: Add ssl certificate key file
+  copy:
+    src: private_files/{{inventory_hostname}}/{{httpd.key}}
+    dest: "{{cert_dir}}/{{httpd.key}}"
+  when: httpd.key is defined
```

Comment:
What do you think about the idea of adding the following parameters:
 * [ ] `owner`
 * [ ] `group`
 * [ ] `mode`
 * [ ] `backup`


For example, we do this in the following roles:
* https://github.com/ARGOeu/argo-ansible/blob/ansible2.9/roles/commons/tasks/cert.yml
* https://github.com/ARGOeu/argo-ansible/blob/ansible2.9/roles/ams/tasks/deploy.yml#L9-L33
* https://github.com/ARGOeu/argo-ansible/blob/ansible2.9/roles/alerta/tasks/deploy.yml#L279-L287
* https://github.com/ARGOeu/argo-ansible/blob/ansible2.9/roles/monbox/tasks/config.yml#L140-L145

Reply:
resolving since this file was replaced by the common role 
roles/common/tasks/cert.yml